dojo/jira_link/helper.py

```diff
@@ -1623,7 +1623,7 @@ def process_resolution_from_jira(finding, resolution_id, resolution_name, assign
                     owner=finding.reporter,
                 )
                 finding.test.engagement.risk_acceptance.add(ra)
-                ra_helper.add_findings_to_risk_acceptance(ra, [finding])
+                ra_helper.add_findings_to_risk_acceptance(None, ra, [finding])
```

Comment:
Should this maybe include the `JIRA` user rather than `None`, as with the `mitigated_by` field in the `else` block [below](https://github.com/DefectDojo/django-DefectDojo/pull/10934/files#diff-ce808f9fd6f829add9bca414ba21800d5b7b333803aaeaff4ff600085c1409f1R1645)? The helper methods don't post a note if `None` is supplied as the user.

Reply:
A note requires a DefectDojo User object that exists in the database in order to be created. We cannot always guarantee there is user that matches the one from jira. 

I suppose we could copy the functionality used in the webhook of creating a dedicated jira service account: https://github.com/DefectDojo/django-DefectDojo/blob/e4fbfe0f2c48c10dcfcd6422b907050a91b22fca/dojo/jira_link/views.py#L254